.github/workflows/automationTemplate.yaml

```diff
@@ -1,6 +1,9 @@
 name: Oryx-SDK-Runtime-Automation-Template
 
 on:
+  push:
+    branches:
+      - 'williamhe/**'
```

Comment:
Need this for a little longer to help me rollout new versions to sandbox storage account